experiments/remhos/openmp/ramble.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+# 
+# SPDX-License-Identifier: Apache-2.0
+ramble:
+  include:
+  - ./configs/software.yaml
+  - ./configs/variables.yaml
+  - ./configs/modifier.yaml
+  config:
+    deprecated: true
+    spack_flags:
+      install: '--add --keep-stage'
+      concretize: '-U -f'
+
+  modifiers:
+  - name: allocation
+
+  applications:
+    remhos:
+      workloads:
+        remhos:
+          experiments:
+             execute_{n_nodes}:
+              variants: 
+                package_manager: spack  
+              variables:
+                n_nodes: ['1','2']
+  software:
+    packages:
+      hypre:
+        pkg_spec: hypre
+        compiler: compiler-gcc
```

Comment:
@august-knox Does the hypre build specifically depend on the gcc compiler for this code?